rfcs/text/0000-Managed-Access-Data.md

```diff
@@ -0,0 +1,346 @@
+can be openly released. I want this to support the aim to build an atlas
+of all human cells.
+
+As a researcher with a pipette, I would like to manage a Data Access
+Committee (DAC) so that I can control access to individuals and groups
+as follows: (NOTE: this is the researcher-run DAC model, which is the
+least burden to the DCP, but the highest burden to the researcher)
+
+1.  (Submitter/Wrangler role) I would like to grant or revoke read-write
+    > access to individuals or groups for data submission purposes
+
+2.  (Consumer role) I would like to grant or revoke read-only access to
+    > individuals or groups based on the type of use they are requesting
+
+3.  (Admin role) I would like to grant or revoke read-write access to
+    > individuals or groups for data management purposes (including
+    > possible redaction of data)
+
+The objects being controlled are files, bundles, and collections, as
+well as setting permissions for all objects related to a project.
+
+As a researcher with a pipette, I would like to set an embargo for
+access to my data so that I may finish my research and publish results
+based on my data before others. I would like to restrict access to my
+data for a set period of time.
+
+**Scientific \"guardrails\"** 
+------------------------------
+
+This plan will be reviewed by members of the [Compliance working
+group]([[https://github.com/HumanCellAtlas/dcp-community/blob/master/charters/Compliance-WG/charter.md]](https://github.com/HumanCellAtlas/dcp-community/blob/master/charters/Compliance-WG/charter.md))
+who will ensure the proposed design meets their guidelines for
+responsible handling of managed access genomic data.
+
+**Detailed Design**
+-------------------
+
+A tiered access model:
+
+**Open Access**: The HCA project heavily favors open-access data. This
+is the primary use case for the DCP. Users will not be required to log
+in to gain access to any openly consented data. Searches executed for
+unauthenticated users will only return open access data.
+
+**Controlled Access**: To access controlled data the user will have to
+log in to the system to identify themselves. Authentication will happen
+via OIDC mechanisms. Once authenticated, the user's searches will return
+open access data as well as controlled access data to which they have
+access.
+
+**Controlled Objects:** Access can be controlled on files, bundles and
+collections. Individuals and groups will be granted access via DAC
+approval. This process will follow the DUOS model (see link below) being
+defined by GA4GH where the types of uses are categorized. This allows
+the DAC to answer some or all access requests programmatically.
+
+**Not supported:** We will not provide a notification mechanism to let
+users know when permissions have been changed.
+
+**High level design of Authorization:** As stated above, all access to
+controlled data will require the user to be authenticated first. This
```

Comment:
The DCP authentication and authorization system is designed to allow federation with other OIDC providers by way of OIDC and SAML federation in Auth0. We designed it specifically with this use case in mind.

Reply:
it would be good to make this explicit in the RFC.   This does make a restriction on what data the DCP can take. To the best of my knowledge, this is a very reasonable restriction.

Reply:
Added content in the Controlled Access section above.